src/strategies_and_params/poststepstrategy.jl

```diff
@@ -113,18 +113,42 @@ function post_step(sc::SignCoherence, replica)
     return (sc.name => coherence(valtype(vector), sc.reference, vector),)
 end
 
+"""
+    num_overlapping(v, w)
+
+Count the number of configurations where both `v` and `w` are nonzero.
+"""
+function num_overlapping(v, w)
+    if length(w) < length(v)
+        v, w = w, v
+    end
+    return mapreduce(+, keys(v); init=0) do k
+        !iszero(localpart(w)[k])
+    end
+end
+
 function coherence(::Type{<:Real}, reference, vector)
     num_correct = mapreduce(+, pairs(vector); init=0) do ((k, v))
         sign(v) == sign(reference[k])
     end
-    return num_correct / length(vector)
+    num_overlap = num_overlapping(reference, vector)
+    if iszero(num_overlap)
+        return 0.0
+    else
+        return 2 * (num_correct / num_overlap) - 1
+    end
 end
 function coherence(::Type{<:Complex}, reference, vector)
     num_correct = mapreduce(+, pairs(vector); init=0 + 0im) do ((k, v))
         ref_sign = sign(reference[k])
         (sign(real(v)) == ref_sign) + im * (sign(imag(v)) == ref_sign)
     end
```

Comment:
Hmm, I don't think that this will do the job for 2Pop complex walkers. E.g. a real walker with `0 - 3im` would be recorded as overlapping and having incorrect sign, while the real population is actually zero. How about instead of recording num_correct, we compute something like
```suggestion
   accumulate = mapreduce(+, pairs(vector); init=0 + 0im) do ((k, v))
        ref_sign = sign(reference[k])
        sign(real(v)) * ref_sign + im * sign(imag(v)) * ref_sign
    end # accumulates values of -1, 0, +1
```